.github/dependabot.yml

```diff
@@ -1,16 +1,19 @@
 version: 2
 updates:
-- package-ecosystem: composer
-  directory: "/"
-  schedule:
-    interval: daily
-    time: "10:00"
-  open-pull-requests-limit: 10
-  reviewers:
-  - c9s
-  - morozov
-  assignees:
-  - morozov
-  labels:
-  - Dependencies
-  versioning-strategy: increase
+    -   package-ecosystem: "composer"
```

Comment:
Some settings are removed. Could you add them back?

Such as `open-pull-requests-limit`, `reviewers` , `assignees` and `labels`.

Reply:
> open-pull-requests-limit

I don't think it's necessary: there won't be that many and in my experience, with a similar config, dependabot will close and re-open MRs so there will not be many in parallel.

>reviewers, assignees

I don't think it's up to date, and is there really a point? However has access can check it out anytime instead.

>labels

The PRs already get `build(deps): ` as a name prefix, if you still want the label we can add it, I just saw no value in it.